actions/apps.ts

```diff
@@ -98,53 +98,45 @@ export function openAppsModal(form: AppForm, call: AppCallRequest): Action {
 }
 
 export function postEphemeralCallResponseForPost(response: AppCallResponse, message: string, post: Post): ActionFunc {
-    return () => {
-        sendEphemeralPost(
+    return (dispatch: DispatchFunc) => {
```

Comment:
You can slightly optimize this by just returning the result of `sendEphemeralPost` post instead of returning a new function. That saves it from calling dispatch twice